### PR TITLE
feat(tilt): wrapper implementation

### PR DESCRIPTION
**Pull request by kurtaqui**

### CODE REVIEW

File: src/tilt.directive.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import * as VanillaTilt from "vanilla-tilt";
+import { OnInit, OnDestroy, Directive, Input, ElementRef, Output, EventEmitter } from "@angular/core";
+import { TiltOptions, TiltValues } from "vanilla-tilt";
+
+@Directive({
+	selector: "[tilt]",
```
Comment (stephenlautier) [inline src/tilt.directive.ts]: for consistency prefix this with `ssv` (i know command is not but i want to rename it)

File: src/tilt.directive.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	exportAs: "ngxTilt",
```
Comment (stephenlautier) [inline src/tilt.directive.ts]: and rename this to `ssvTitlt`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,110 @@
...
+**Quick links**
+
+[Change logs][changeLog] | [Project Repository][projectUri]
+
+# Installation
+
+Get library via [npm]
+
+```bash
+npm install vanilla-tilt && @types/vanilla-tilt && @ssv/ngx.tilt
```
Comment (stephenlautier) [inline README.md]: remove `&&` just separate with spaces

File: README.md [hunk 4]
```diff
@@ -1,2 +1,110 @@
...
+[projectgit]: https://github.com/ssv/ngx.tilt.git
+[changelog]: ./CHANGELOG.md
+[releaseworkflowwiki]: ./docs/RELEASE-WORKFLOW.md
+
+[npm]: https://www.npmjs.com
+
+# @ssv/ngx.tilt
+[![CircleCI](https://circleci.com/gh/ssv/ngx-tilt)](https://circleci.com/gh/ssv/ngx.tilt)
+
+This is an Angular Wrapper for [Vanilla Tilt](https://github.com/micku7zu/vanilla-tilt.js/) a smooth 3D tilt javascript.
```
Comment (stephenlautier) [inline README.md]:
```suggestion
Angular Wrapper for [Vanilla Tilt](https://github.com/micku7zu/vanilla-tilt.js/) a smooth 3D tilt.
```

File: ng-package.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+	"$schema": "./node_modules/ng-packagr/ng-package.schema.json",
+	"deleteDestPath": false,
+	"lib": {
+		"entryFile": "./src/index.ts",
+		"whitelistedNonPeerDependencies": ["."],
+		"umdModuleIds": {
+      "tslib": "tslib",
```
Comment (stephenlautier) [inline ng-package.json:8]: formatting

File: package.json [hunk 6]
```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@ssv/ngx.tilt",
+  "version": "0.1.0",
+  "description": "Angular wrapper for vanilla tilt.",
```
Comment (stephenlautier) [inline package.json]: keep the same desc

> Angular Wrapper for [Vanilla Tilt](https://github.com/micku7zu/vanilla-tilt.js/) a smooth 3D tilt.

File: package.json [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  "author": "Kurt Aquilina <kaquilina92@gmail.com>",
+  "license": "MIT",
+  "private": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ssv/ngx.tilt"
+  },
+  "peerDependencies": {
+    "@angular/core": ">=6.0.0",
+    "@types/vanilla-tilt": "^1.4.1",
```
Comment (stephenlautier) [inline package.json:35]: i dont think this is needed no? if so remove from readme installs aswell

File: src/tilt.directive.ts [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import * as VanillaTilt from "vanilla-tilt";
+import { OnInit, OnDestroy, Directive, Input, ElementRef, Output, EventEmitter } from "@angular/core";
+import { TiltOptions, TiltValues } from "vanilla-tilt";
+
+@Directive({
+	selector: "[ssv-tilt]",
```
Comment (stephenlautier) [inline src/tilt.directive.ts]: should be camelCased not kebabed (to keep with how angular name things)

File: ng-package.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "./node_modules/ng-packagr/ng-package.schema.json",
+  "deleteDestPath": false,
+  "lib": {
+    "entryFile": "./src/index.ts",
+    "whitelistedNonPeerDependencies": ["."],
+    "umdModuleIds": {
+      "tslib": "tslib",
+      "lodash": "_"
```
Comment (stephenlautier) [inline ng-package.json]: you might need to add `vanilla-tilt: vanillaTilt` in here (see warnings when you build)

File: src/tilt.directive.ts [hunk 10]
```diff
@@ -0,0 +1,45 @@
+import * as VanillaTilt from "vanilla-tilt";
```
Comment (stephenlautier) [inline src/tilt.directive.ts]: seeing the usage syntax i think you should use

```ts
import VanillaTilt from "vanilla-tilt"; (and remove `.default` from usage below)
```

File: package.json (hunk 7 above)
Comment (kurtaqui) [reply package.json:35]: it is since if you need types for example `TiltOptions` unless we re-export

File: package.json [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+  "name": "@ssv/ngx.tilt",
+  "version": "0.1.0",
+  "description": "Angular Wrapper for Vanilla Tilt a smooth 3D tilt.",
+  "keywords": [
+    "ngx",
+    "angular2",
+    "angular",
+    "tilt",
+    "ux",
+    "ui"
```
Comment (stephenlautier) [inline package.json:12]: add `tilt` and `vanilla tilt`

File: README.md [hunk 12]
```diff
@@ -1,2 +1,110 @@
...
+  ]
+}
+export class AppModule {
+}
+```
+
+```html
+<!-- simple -->
+<div ssv-tilt>
+  show only when large
```
Comment (stephenlautier) [inline README.md]:
```suggestion
  im tilt
```

**Review by stephenlautier — APPROVED** [review]
looks good, thanks!

**Pull request merged**